src/handlers/pullRequest.ts

```diff
@@ -36,4 +36,28 @@ export default (app: Probot) => {
       await pr.clearLabels();
     }
   });
+
+  app.on(['pull_request.converted_to_draft'], async context => {
+    const pr = new PullRequest(context);
+
+    let title = pr.data.title;
+    if (pr.wip) {
```

Comment:
This is probably unnecessary, and might be confusing to the author. If it already had WIP, and converting it to draft made that go away, they might think the bot is confused.